lib/node_modules/@stdlib/constants/float32/half-pi/include/stdlib/constants/float64/half_pi.h

```diff
@@ -0,0 +1,27 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#ifndef STDLIB_CONSTANTS_FLOAT32_HALF_PI_H
+#define STDLIB_CONSTANTS_FLOAT32_HALF_PI_H
+
+/**
+* Macro for one half times the mathematical constant π.
+*/
+#define STDLIB_CONSTANT_FLOAT32_HALF_PI 1.5707963
```

Comment:
Notice the trailing `f`: https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/constants/float32/smallest-normal/include/stdlib/constants/float32/smallest_normal.h#L25. I suggest including one more digit than what is supported by single-precision. A C compiler will ensure correctly rounded.